app/code/core/Mage/Adminhtml/Block/Customer/Edit/Renderer/Newpass.php

```diff
@@ -49,6 +49,7 @@ public function render(Varien_Data_Form_Element_Abstract $element)
         if ($element->getNote()) {
             $html .= '<p class="note"><span>' . $element->getNote() . '</span></p>';
         }
+        $html .= '<p id="email-passowrd-warning" style="display:none;" class="note"><span>' . Mage::helper('customer')->__('Warning: email cotains password in plaintext will be sent.') . '</span></p>';
```

Comment:
this isn't true because a link is sent instead of a password?

Reply:
There's also a typo: `cotains`

Reply:
> IMO the automatic email for a manually set password should be removed (if the manual password set feature is retained).

Hmmm... but it's hard to know if a use case exists where it's necessary to send a plaintext password to customer.

Anyway, the objective of this PR is to provide the UI for the backend admin to email a link to set password, especially for customer accounts created in the backend. The automatic email is an existng feature that can be addressed separetely, I don't want to do too many things.